src/main/resources/application.conf

```diff
@@ -554,7 +554,6 @@ mantis {
       # - kevm: runs a binary provided at `executable-path` with `port` and `host` as arguments (`./executable-path $port $host`)
       # - mantis: if `executable-path` is provided, it will run the binary with `port` and `host` as arguments
       #           otherwise mantis VM will be run in the same process, but acting as an external VM (listening at `host` and `port`)
-      # - none: doesn't run anything, expect the VM to be started by other means
```

Comment:
We should update _all_ comments in this section

Reply:
I think the remaining comments are still valid.

Reply:
The comments suggest that setting type to `kevm` or `iele` runs the binary. It's not clear that it's actually the `exectuable-path` that determines whether the binary is run or not